### DISCUSSION TITLE
腳踏車車位問題

### DISCUSSION THREAD

**Discussion by CeBru13** [1101-GS4719-C/Python-Community-GS4719-C#2249] [General]
在學校常常會看到一堆腳踏車擠在一起，或是會有車停在被禁止停車的地方，EX.男13、教研大樓、志道樓之類的，有人有什麼想法嗎？

**Comment by Chattenoire** [comment]
基本上是學校的動線規劃不佳，加上腳踏車位不足或是在不對的地方，所以大家只能亂停
(example: 教研大樓1樓)
老實說這學校很多問題都不是(根本上)源自學生，而是行政人員的規劃，嘻嘻

**Comment by qiu0109** [comment]
有時候沒有腳踏車車位時，真的就只能亂停車。而且在學校常常看到一些車子都生鏽的很誇張，還長蜘蛛網，看起來就是畢業學長姐留下的，我覺得學校應該清理一下這些腳踏車，讓車位空出來給學生們使用。

**Comment by christaaah** [comment]
我們系館平常的時候還好，但是遇到所有年級都有課的時段再加上前一堂還有其他的課，那些設置的停車的車位完全不夠，導致道路附近停了很多腳踏車，我覺得這是不可避免的。如果不擋到行人或車輛通行，並且只是暫時停在那裡的話我覺得就還好。

**Comment by wcyeee** [comment]
我也覺得問題可能是有很多被棄置的腳踏車還放在原地。因為現在仔細看腳踏車停車位有很多看起來就是已經被拋棄很久的車，這些車不佔少數，所以有很多位置也被他們佔走，我覺得如果可以把這些車子移開的話應該這個狀況會有所改善。

**Comment by ShuliANN** [comment]
我也發現，許多看起來早已被拋棄的腳踏車，長期佔據了大量空間。而之所以如此，主要是因為中央的腳踏車管理規定，只要車子貼有標示，就不會被清理。結果是這些貼了標示的舊車一再被放置在原地，無人處理。應該可以跟學校建議，針對這些車子來處理，改善這個狀況。

**Comment by yue083139** [comment]
校園裡的廢棄腳踏車真的很多，有些很明顯就是丟在那裡沒人在用的，但因為有貼了標籤就沒人管，感覺貼標應該要1-2年換一次，過期證照的腳踏車就照常清掉，然後就是宿舍多的地方也容易會有車位不夠用的問題，雖然這個我也暫時想不到什麼解決辦法就是了，只能每次停都像在開彩票。

**Comment by BOSENCHANG2004** [comment]
我認為應該透過學生會發起聯署，讓學校去正視這件事情。應當透過辨識腳踏車證來確定此車的主人是否還在學校，如果此車已經占用在一個位置許久，且其主人早已經離開學校，則應定期強制執行把廢棄的腳踏車報廢。

**Comment by a414414** [comment]
我覺得學校應該要定期清除廢棄腳踏車，並且要做確實。
以前住在男13也看到很多看起來就沒人在騎，不知道到底放了多久的腳踏車，車況也是相當悽慘，
鏈子生鏽、鍊子脫落，甚至看起來不用想都知道沒辦法騎的，但也都沒被清除掉。
其實學校會在寒暑假清理停在校內，且未貼車牌的腳踏車，但我認為學校進行清除的範圍應該不夠大，有很多地方都還是有廢棄的腳踏車。
像今天騎來志希館上課，在腳踏車停車位就看到一堆生鏽、長灰塵的腳踏車，就應該是要被清除掉的，根本沒人騎。

**Comment by Kaz9487** [comment]
學校平常根本不會認真重視停車位不足的事，就算有外賓也只是那段時間會禁停跟移車，沒有認真去想過為甚麼每天都有一大堆腳踏車臨停的問題。而且，有些腳踏車都放都輪胎被樹葉樹枝淹過去了也不清掉，占掉寶貴的停車空間。
